docs/source/reference/kubernetes/kubernetes-setup.rst

```diff
@@ -13,195 +13,140 @@ Kubernetes Cluster Setup
+        For on-prem deployments with kubeadm, RKE2, K3s or other distributions.
 
-   .. code-block:: console
+    .. grid-item-card:: Google Cloud - GKE
+        :link: kubernetes-setup-gke
+        :link-type: ref
+        :text-align: center
 
-     $ gcloud container clusters get-credentials <cluster-name> --region <region>
+        Google's hosted Kubernetes service.
 
-     # Example:
-     # gcloud container clusters get-credentials testcluster --region us-central1-c
+    .. grid-item-card::  Amazon - EKS
+        :link: kubernetes-setup-eks
+        :link-type: ref
+        :text-align: center
 
-3. [If using GPUs] If your GKE nodes have GPUs, you may need to to
-   `manually install <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/>`_
-   nvidia drivers. You can do so by deploying the daemonset
-   depending on the GPU and OS on your nodes:
+        Amazon's hosted Kubernetes service.
 
-   .. code-block:: console
 
-     # For Container Optimized OS (COS) based nodes with GPUs other than Nvidia L4 (e.g., V100, A100, ...):
-     $ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded.yaml
+.. _kubernetes-setup-gpusupport:
 
-     # For Container Optimized OS (COS) based nodes with L4 GPUs:
-     $ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded-latest.yaml
+Step 2 - Set up GPU support
+---------------------------
 
-     # For Ubuntu based nodes with GPUs other than Nvidia L4 (e.g., V100, A100, ...):
-     $ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/ubuntu/daemonset-preloaded.yaml
+To utilize GPUs on Kubernetes, your cluster must:
 
-     # For Ubuntu based nodes with L4 GPUs:
-     $ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/ubuntu/daemonset-preloaded-R525.yaml
+1. Have the ``nvidia.com/gpu`` **resource** available on all GPU nodes and have ``nvidia`` as the default runtime for your container engine. See `Nvidia GPU Operator Installation Guide <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#install-nvidia-gpu-operator>`_ for more details.
+2. Have a **label on each node specifying the GPU type**. See :ref:`Setting up GPU labels <kubernetes-gpu-labels>` for more details.
 
-   To verify if GPU drivers are set up, run ``kubectl describe nodes`` and verify that ``nvidia.com/gpu`` is listed under the ``Capacity`` section.
 
-4. Verify your kubeconfig (and GPU support, if available) is correctly set up by running :code:`sky check`:
+.. tip::
```

Comment:
Can we move this tip right after number 1?

Reply:
I'm leaning towards keeping the tips/notes after the list of requirements, otherwise it gets hard to read what all needs to be done to setup GPU support.

Reply:
That makes sense! Then can we add a sentence in number 1 to say that clusters setup with the deployment instructions do not need to setup the gpu operator? So a user does not need to go through the nvidia doc only to find out it is not needed for their cluster?

Also, we may want to mention that the instructions of gpu operators setup are provided in the deployment doc, as the nvidia doc might be too long to follow.

Reply:
Good point - added a sub bullet. does that help?